configs/community.dbuild

```diff
@@ -411,1940 +222,28 @@ build += {
-      "akka-http-circe"
-      // library com.sksamuel.avro4s#avro4s-json is not provided
-      "akka-http-avro4s"
-      // library com.avsystem.commons#commons-core is not provided
-      "akka-http-avsystem-gencodec"
-    ]
-    extra.options: ["-Dbintray.user=dummy", "-Dbintray.pass=dummy"]
-    // apparently some dependency dropped its scala-reflect dependency?
-    // and we have the newer version of that dependency here in the
-    // community build, while the akka-http-jackson subproject's
-    // declared dependency is on the old version?
-    // that's my best guess of why we need this here, anyway
-    deps.inject: ${vars.base.deps.inject} [
-      "org.scala-lang#scala-reflect"
-    ]
-    extra.commands: ${vars.default-commands} [
-      "set every bintrayReleaseOnPublish := false"
-      "set libraryDependencies in `akka-http-jackson` += \"org.scala-lang\" %% \"scala-reflect\" % \""${vars.scala-version}"\""
-    ]
-  }
-
-  // frozen (February 2018) at an October 2017 commit; see
-  // https://github.com/scala/community-builds/issues/682
-  ${vars.base} {
-    name: "scaladex"
-    uri:  ${vars.uris.scaladex-uri}
-    extra.exclude: [
-      // no Scala.js plz
-      "sharedJS", "client"
-      // "java.io.IOException: Cannot run program "sass": error=2, No such file or directory"
-      // not insurmountable -- we could add commands to install the gem in the current directory,
-      // and then customize `sassExecutable in Assets` if needed. but I feel it's not really worth
-      // the additional labor and (more importantly) the additional fragility
-      "server"
-    ]
-  }
-
-  ${vars.base} {
-    name: "scala-debugger"
-    uri:  ${vars.uris.scala-debugger-uri}
-    extra.exclude: [
-      // no sbt plugins plz!
-      "sbtScalaDebuggerPlugin"
-      // Missing dependency: the library org.senkbeil#grus-layouts
-      "scalaDebuggerDocs"
-      // depends on ammonite, which we no longer have
-      "scalaDebuggerTool"
-    ]
-    // java.util.concurrent.ExecutionException: java.lang.OutOfMemoryError: GC overhead limit exceeded
-    // perhaps only the -Xmx is really necessary, but let's try it with all of the repo's own .jvmopts:
-    extra.options: ["-Xms1g", "-Xmx4g", "-Xss2m", "-XX:MaxMetaspaceSize=256m"]
-  }
-
-]}
-
-//// space: jawn_0_11
-
-build += {
-
-  space: scala.main.jawn_0_11
-
-  check-missing: [ true, false ]
-  cross-version: [ disabled, standard ]
-  extraction-version: ${vars.scala-version}
-  sbt-version: ${vars.sbt-version}
-
-  projects: [
-
-  // frozen (April 2018) at an April 2018 commit just before an upgrade to jawn 0.12.
-  // at some point we should move everything off 0.11 and onto 0.12, but there's no rush.
-  ${vars.base} {
-    name: "circe"
-    uri:  ${vars.uris.circe-uri}
-    extra.sbt-version: ${vars.sbt-1-version}
-    extra.projects: [
-      // easy
-      "coreJVM", "numbersJVM"
-      // harder
-      "jawn"
-      // bunch more stuff that all depends on jawn
-      "parserJ
```

Comment:
Can I ask, what's the reason for excluding these? Is it to save build time?

Reply:
this is an include list, not an exclude list. `extra.projects` is include, `extra.exclude` is exclude.

typically we include what downstream projects actually depend on, and then the rest is gravy (we include or not based on some estimate of cost/benefit. most often the cost has to do with maintenance effort, not build time)

Reply:
(btw I'm always happy, if someone asks, to add a comment to the config file clarifying a decision of this sort. you'll see there are already many such comments, but also many places where they are missing.)

Reply:
Ok cool thanks for clarifying. Can I confirm also that `test:compile` is run on the included projects? If it is then exclusion of the `gh-pages` module here is fine but if not; well preferably we'd switch it on for scalajs-react but if that's unfeasible for whatever reason then enabling `gh-pages` module would be the second-most valuable thing.

Reply:
> Can I confirm also that `test:compile` is run on the included projects?

not only that, but `test:test`, too

Reply:
Awesome! Cheers